instances/anatomicalPlane/coronal.jsonld

```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/anatomicalPlane/coronal",
   "@type": "https://openminds.ebrains.eu/controlledTerms/AnatomicalPlane",
-  "definition": "An anatomical plane which divides the body into divides the body into dorsal and ventral (back and front, or posterior and anterior) parts.",
+  "definition": "An anatomical plane which divides the body or parts of the body into rostral and caudal (anterior and posterior, or front and back) parts.",
```

Comment:
Rename to frontal plane? or add this only as synonym?

Suggestion: The 'coronal plane' divides the body and extremities of an organism into dorsal and ventral parts and an organ into anterior and posterior parts.

(I'm not happy with that one)

Reply:
I agree this should be renamed to frontal.

Suggestion:
The 'frontal plane' is always perpendicular to the axial and longitudinal axes of an organism, extremity or organ, dividing it into an anterior, rostral or front part, and a posterior, caudal or back part.

Link to webpage: https://www.healthline.com/health/body-planes#coronal-plane
We could use this as a reference too, but I don't know: https://www.frontiersin.org/articles/10.3389/fninf.2023.1154080/full


Could we add synonym?

"synonym": [
"Coronal"
]